src/fiddle.ts

```diff
@@ -31,6 +32,10 @@ export class Fiddle {
  */
 export type FiddleSource = Fiddle | string | Iterable<[string, string]>;
 
+export interface CreateOptions {
```

Comment:
To match the existing naming convention (e.g. `ElectronVersionsCreateOptions`), this should be named `FiddleFactoryCreateOptions`.